Doc/library/lzma.rst

```diff
@@ -311,6 +311,25 @@ Compressing and decompressing data in memory
    *preset* and *filters* arguments.
 
 
+.. function:: crc32(data[, value])
+
+   .. index::
+      single: Cyclic Redundancy Check
+      single: checksum; Cyclic Redundancy Check
+
+   Computes a CRC (Cyclic Redundancy Check) checksum of *data*. The
+   result is an unsigned 32-bit integer. If *value* is present, it is used
+   as the starting value of the checksum; otherwise, a default value of 0
+   is used.  Passing in *value* allows computing a running checksum over the
+   concatenation of several inputs.  The algorithm is not cryptographically
+   strong, and should not be used for authentication or digital signatures.  Since
+   the algorithm is designed for use as a checksum algorithm, it is not suitable
+   for use as a general hash algorithm.
+
+   .. versionadded:: 3.14
+      The result is always unsigned.
```

Comment:
This seems strange, isn’t the versionadded directive without body enough here? (meaning the whole function is the addition)

Reply:
```suggestion
```